pontoon/base/templates/translate.html

```diff
@@ -173,7 +173,7 @@
 
             {{ Filter.item('untranslated', 'Untranslated') }}
             {{ Filter.item('unchanged', 'Unchanged') }}
-            {{ Filter.item('has-suggestions', 'Has Suggestions') }}
+            {{ Filter.item('has-suggestions', 'Has Unreviewed Suggestions') }}
```

Comment:
Changing the slug will break compatibility for bookmarks and links. We should keep this is mind if we do change it. Maybe keep an alias of the old value?